boule/_triaxialellipsoid.py

```diff
@@ -222,6 +227,25 @@ def semiaxes_mean_radius(self):
         """
         return (self.semimajor_axis + self.semimedium_axis + self.semiminor_axis) / 3
 
+    @property
+    def area(self):
+        r"""
+        The area of the ellipsoid.
+        Definition: :math:`A`.
+        Units: :math:`m^2`.
+        """
+        # see https://en.wikipedia.org/wiki/Ellipsoid#Surface_area
+        a = self.semimajor_axis
+        b = self.semimedium_axis
+        c = self.semiminor_axis
+        return 2 * np.pi * c**2 + 2 * np.pi * a * b * (
+            elliprf((c / a) ** 2, (c / b) ** 2, 1)
+            - (1 / 3)
+            * (1 - (c / a) ** 2)
+            * (1 - (c / b) ** 2)
+            * elliprd((c / a) ** 2, (c / b) ** 2, 1)
+        )
```

Comment:
@MarkWieczorek I was wondering how you arrived at this expression from the one in the Wikipedia link. Scipy does have the incomplete elliptic integrals of first and second kind. So why use this version?

Reply:
I hate citing wikipedia. I also hate using it as a primary source like this, but it is only the area, so what could go wrong?

So if you go one edit earlier, you find the above equation: https://en.wikipedia.org/w/index.php?title=Ellipsoid&oldid=1219229944

Then there was an edit yesterday that says "Use simpler expression of the area in terms of Carlson symmetric integrals." The new expression only uses elliprg, instead of elliprf and elliprd. The new equation looks more elegant, so like me change it and check that we get the same answer.

Reply:
As to why use that expression, the old wiki page stated "Unlike the expression with F(φ, k) and E(φ, k), the variant based on the Carlson symmetric integrals yields valid results for a sphere and only the axis c must be the smallest, the order between the two larger axes, a and b can be arbitrary."

Reply:
😆 yeah I know what you mean! Thanks for the explanation! I changed it to use the RG version and tests pass. I'll push that here and merge.

Reply:
You beat me to it. It turns out that the order of the three arguments to the elliprg function doesn't matter, so this is much better.